Makefile

```diff
@@ -27,9 +28,9 @@ windows: mkbin generate
 
 generate: version-file ## updates version of pipeline-as-code and cli in pkg/version file
 version-file:
-	echo '{"pac": "$(PAC_VERSION)", "tkn": "$(TKN_VERSION)", "opc": "$(OPC_VERSION)"}' > pkg/version.json
+	echo '{"pac": "$(PAC_VERSION)", "tkn": "$(TKN_VERSION)", "results": "$(RESULTS_VERSION)", "opc": "$(OPC_VERSION)"}' > pkg/version.json
 
-version-updates: ## updates pipeline-as-code and cli version in go.mod
+version-updates: ## updates pipeline-as-code, cli and results version in go.mod
```

Comment:
But we aren't updating the version here. I don't see an update command for results under this Target.

Reply:
fixed